guidelines_for_grant_submissions.md

```diff
@@ -31,4 +31,15 @@ In contrast, proposals which are unlikely to be accepted include:
 * Development of hardware or software which will not be released under an open licence
 * Applications which lack justification and explanation of the amount requested
 
+For grants that include the funding of hardware purchases to support Open Source software development:
+
+* The grant application should include a description of what the project aims to achieve.  For example:
+    - Support for this hardware is committed to the upstream project repository.
+    - A sustainable community of developers is formed to continue development and support of the hardware.
+    - An individual takes an active role in the development of Open Source support for the hardware, which would have otherwise been impossible due to the cost of the hardware.
+
+* An individual or group may receive at most one grant of this nature per grant cycle.
+
+* In the project's final report provided to Linux Australia, include a review of what went well and what did not. If the project’s objectives were not met, explain what might have helped the project achieve more.  If applicable, describe any follow up activities which could build on the progress made and how others might be able to assist to provide additional value for the broader community.
```

Comment:
Should we ask the project to specify/describe what happened to the hardware, or will happen to it, e.g.:

* To the extent possible and this question applies, can the hardware be recycled if it is destroyed/used up during the project lifetime?
* To the extent possible and this question applies, will the hardware continue to be used, and if so what is its expected lifetime?
* To the extent possible and this question applies, does the organisation want to stipulate anything about suppliers? Probably not, I would say, but if we were Emit No CO2 Inc, say, we'd probably stipulate that hardware be sourced from a carbon neutral source if possible.

Reply:
Thank you for raising these points. Having some visibility into the hardware's future treatment would be instructive. Given the wide range of hardware in existence now and the diverse way it is applied, I expect it would be difficult to appropriately cover all possibilities in a succint manner. The concern is that in trying to do so we would be adding more length to a document which is already at the limit of what seems reasonable. That said, including a request to indicate where the hardware ends up at the conclusion of the project's scope could be possible.

On the question of supplies, I don't believe that's within LA's scope, which is in line with your suspicions. Often hardware is only available from particular vendors, making stipulations somewhat pointless.

Reply:
Commit f6afa31 adds addresses the concern about the future of hardware funded by grants in line with the above suggestion.